snippets/cpp/VS_Snippets_CLR/InstallContext_InstallContext/CPP/installcontext_installcontext.cpp

```diff
@@ -94,7 +94,7 @@ int main()
    {
       // <Snippet2>
       // There are no command line arguments, create an empty 'InstallContext'.
-      myInstallObject->myInstallContext = gcnew InstallContext;
+      myInstallObject->Context = gcnew InstallContext;
```

Comment:
I'm not sure this is correct. InstallContext_Example defines a field called myInstallContext on line 35.